### PR TITLE
Update to Scala 2.13.8

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: .github/workflows/integration-tests-grpc.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - { java-version: adopt@1.8,      scala-version: 2.13.3,  sbt-opts: '' }
-          - { java-version: adopt@1.11.0-9, scala-version: 2.13.3,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
+          - { java-version: adopt@1.8,      scala-version: 2.13.x,  sbt-opts: '' }
+          - { java-version: adopt@1.11, scala-version: 2.13.x,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
```
Comment (octonato) [inline .github/workflows/integration-tests-grpc.yml:19]: why we use this `2.13.x` syntax here and below we removed the scala-version?

Comment (octonato) [inline .github/workflows/integration-tests-grpc.yml:19]:
```suggestion
          - { java-version: adopt@1.8,  scala-version: 2.13.x,  sbt-opts: '' }
          - { java-version: adopt@1.11, scala-version: 2.13.x,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
```

Am I being a formatting freak? 
(feel free to ignore it 😄 )

**Review by octonato — APPROVED** [review]
LGTM, but I have a question and a formatting suggestion (that can be ignored)

File: .github/workflows/integration-tests-grpc.yml (hunk 1 above)
Comment (patriknw) [reply .github/workflows/integration-tests-grpc.yml]: it was some leftover, I have removed in https://github.com/akka/akka-projection/pull/687/commits/7457aa2b6dea68915e6596495f4f20338992b0a3

**Pull request merged**